pulp_smash/tests/pulp3/utils.py

```diff
@@ -289,3 +287,17 @@ def clean_content_units(cfg=None):
     client = api.Client(cfg, api.json_handler)
     for content_unit in client.get(FILE_CONTENT_PATH)['results']:
         client.delete(content_unit['_href'])
+
+
+def get_repo_versions(repo):
+    """Return hrefs of repository versions.
+
+    :param repo: A dict of information about the repository.
+    :returns: A sorted list with the hrefs of repository versions.
+    """
+    client = api.Client(config.get_config(), api.json_handler)
+    versions = client.get(repo['_versions_href'])['results']
```

Comment:
This works well so long as only one page of results are returned. If there are more repository versions than can fit in a single page, this method will return incorrect results. This is OK in the sort term, but a generator that walks through all pages and yields hrefs should be written.

Reply:
@Ichimonji10, that is a great improvement. I will address your comment.